app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java

```diff
@@ -49,7 +49,7 @@
 
     SharedPreferences.Editor editor;
     String timestamp;
-    Fragment fragment = null;
+    public Fragment activeFragment = null;
```

Comment:
@georgezsiga why not use `BaseFragment` type, then the casting wont be necessery on line 97